libc/optee/abort.c

```diff
@@ -3,4 +3,6 @@
 
 void abort(void)
 {
+    while (1)
```

Comment:
won't this cause a hang instead of an abort?

Reply:
Yes. Now that TA's are being built in master, the plan is to add the host-side functionality to load TA's from Linux. Once TA's load, I'll add the remaining missing functionality inside the TA. I've added an infinite loop here so that I can catch it during debugging if it ever hits. The proper implementation will be to call into OP-TEE's abort handler.

Reply:
Please add a TODO comment inline

Reply:
Added.